### PR TITLE
16 edit lists title

### LINKED ISSUE #16: Allow The User to Edit a List's Title
At the current moment, the user has no way of editing the title of a list after creating one. This functionality should be added.

### PR DESCRIPTION
**Pull request by tycebrown**
Added functionality to edit a list's title.

Points/considerations/questions:

  - Is the constructor `makeEditButton()` useless, duplicate code? 
  - Anything else need restyling, renaming, restructuring?

### CODE REVIEW

**Comment by tycebrown** [comment]
Somehow dependent on issue #15 and PR #22 (search in single list).

File: lib/view/editing_actions_modal_bottom_sheet.dart [hunk 1]
```diff
@@ -38,19 +38,27 @@ class EditingActionButton extends StatelessWidget {
   });
 
   factory EditingActionButton.makeDeleteButton(
```
Comment (Pertempto) [inline lib/view/editing_actions_modal_bottom_sheet.dart]:
```suggestion
  factory EditingActionButton.deleteButton(
```

File: lib/view/editing_actions_modal_bottom_sheet.dart [hunk 2]
```diff
@@ -38,19 +38,27 @@ class EditingActionButton extends StatelessWidget {
...
-          {required final void Function() onDelete}) =>
+          {required void Function() onDelete}) =>
       EditingActionButton(
           onPressed: onDelete,
           icon: Icons.delete,
           label: 'Delete',
           color: Colors.red);
 
+  factory EditingActionButton.makeEditButton(
```
Comment (Pertempto) [inline lib/view/editing_actions_modal_bottom_sheet.dart]:
```suggestion
  factory EditingActionButton.editButton(
```

**Review by Pertempto — CHANGES_REQUESTED** [review]
Looking pretty good! I don't like how the editing actions buttons are spread across with `MainAxisAlignment.spaceBetween`. I think they would look better all on the left side. Use a `Wrap` with the `spacing` parameter set to something like 8, 12, or 16.

**Comment by Pertempto** [comment]
@tycebrown For future reference, please let the reviewer mark the threads as resolved. That way, they can double check that they were handled correctly.

**Review by Pertempto — APPROVED** [review]

**Comment by tycebrown** [comment]
> @tycebrown For future reference, please let the reviewer mark the threads as resolved. That way, they can double check that they were handled correctly.

Oh, okay, will keep that in mind.

**Pull request merged**